subworkflows/local/tumor_variant_calling.nf

```diff
@@ -29,6 +30,7 @@ workflow TUMOR_ONLY_VARIANT_CALLING {
         panel_of_normals_tbi          // channel: [optional]  panel_of_normals_tbi
         chr_files
         mappability
+        bwa                           // channel: [optional] bwa
```

Comment:
Can the reference input be listed alphabetically? (you'll need to reorganize when calling this subworkflow too)

Reply:
Oh of course! The other two varcalling subworkflows are not alphabetical, either, should I sort those as well?

Reply:
cram should stay at the top though, but all the references can be sorted